pkg/event/events.go

```diff
@@ -125,3 +125,37 @@ func NewBackgroundSuccessEvent(policy, rule string, source Source, r *unstructur
 
 	return events
 }
+
+func NewPolicyExceptionEvent(engineResponse *response.EngineResponse, ruleResp *response.RuleResponse) Info {
+	var reasonBuilder, messageBuilder strings.Builder
+	defer reasonBuilder.Reset()
+	defer messageBuilder.Reset()
+
+	exceptionName, exceptionNamespace := getExceptionEventInfoFromRuleResponseMsg(ruleResp.Message)
+
+	fmt.Fprintf(&reasonBuilder, "exception %s/%s matches resource %s", exceptionNamespace, exceptionName, engineResponse.PatchedResource.GetName())
+	fmt.Fprintf(&reasonBuilder, "resource %s is excepted from rule %s due to exception %s/%s", engineResponse.PatchedResource.GetName(), ruleResp.Name, exceptionNamespace, exceptionName)
```

Comment:
`excepted` is not a very commonly use word. I suggest phrasing as:

```
resource %s was skipped from rule %s due to policy exception %s/%s"
```
or

```
rules %s was not applied to resource %s due to policy exception %s/%s"
```

Reply:
Thx for the phrasing. Looks nicer!